src/mango_native_proc.erl

```diff
@@ -227,6 +238,10 @@ get_default_enabled(Props) ->
     end.
 
 
+get_index_array_lengths(Props) ->
+    couch_util:get_value(<<"index_array_lengths">>, Props, true).
```

Comment:
...and ensure that index_array_length is a boolean, not "true"/"false" strings or something else.